runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkCachedArtifactPaths.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.beam.runners.flink;
+
+/**
+ * Determines artifact path names within the
+ * {@link org.apache.flink.api.common.cache.DistributedCache}.
+ */
+public class FlinkCachedArtifactPaths {
+  private final static String ARTIFACT_TEMPLATE = "ARTIFACT_%s_%s";
```

Comment:
I think it's appropriate to put these templates in-line since they are never referenced elsewhere.